### ISSUE TITLE
Links surrounding spotlights must be set to display:block to show outlines correctly

### ISSUE THREAD

**Issue by cjg89** [UCF/UCF-Spotlights-Plugin#24]
The `outline` property will fail to wrap correctly around link-wrapped spotlights, due to links being inline elements.  This results in outlines on focus not surrounding spotlights properly; in the case of square spotlights, they're not visible at all.

Examples of current focus states:

Vertical:
![Screen Shot 2019-10-16 at 4 50 43 PM](https://user-images.githubusercontent.com/1649216/66957914-1e12eb00-f035-11e9-9238-42b2c18e9634.png)

Square:
![Screen Shot 2019-10-16 at 4 51 15 PM](https://user-images.githubusercontent.com/1649216/66957955-33881500-f035-11e9-9a39-5ba918fb1f37.png)

The horizontal layout is not affected since that layout doesn't have a link wrapped around it.
